src/PostgREST/Config.hs

```diff
@@ -145,6 +147,13 @@ readOptions = do
       return appConf
 
   where
+    parsedPairToTextPair :: (Name, Value) -> (Text, Text)
+    parsedPairToTextPair (k, v) = (k, newValue)
+      where
+        newValue = case v of
```

Comment:
`newValue` looks similar to functions like `coerceInt` and `coerceBool`. What if you called it `coerceText` and added it near those others?

Then

```haskell
parsedPairToTextPair = id *** coerceText
```
Unless you feel like this last definition is a little too weird looking.

Reply:
I gave up on `id *** coerceText`. If you can apply your haskell fu successfully, I will dispatch my wizard hat forthwith.